assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -396,27 +397,39 @@ function BannerNotification( {
 							</Fragment>
 						) }
 
-						{ ctaLink && (
-							<Button
-								className="googlesitekit-notification__cta"
-								href={ ctaLink }
-								target={ ctaTarget }
-								onClick={ handleCTAClick }
-								disabled={ isAwaitingCTAResponse }
-							>
-								{ ctaLabel }
-							</Button>
-						) }
+						{ ( ctaLink || isDismissible || dismiss ) && (
+							<div className="googlesitekit-publisher-win__actions">
+								{ ctaLink && (
+									<Button
+										className="googlesitekit-notification__cta"
+										href={ ctaLink }
+										target={ ctaTarget }
+										onClick={ handleCTAClick }
+										disabled={ isAwaitingCTAResponse }
+									>
+										{ ctaLabel }
+									</Button>
+								) }
 
-						<Spinner isSaving={ isAwaitingCTAResponse } />
+								<Spinner isSaving={ isAwaitingCTAResponse } />
+
+								{ isDismissible &&
+									dismiss &&
+									! isAwaitingCTAResponse && (
+										<DismissComponent
+											onClick={ handleDismiss }
+										>
+											{ dismiss }
+										</DismissComponent>
+									) }
+							</div>
+						) }
 
-						{ isDismissible &&
-							dismiss &&
-							! isAwaitingCTAResponse && (
-								<DismissComponent onClick={ handleDismiss }>
-									{ dismiss }
-								</DismissComponent>
-							) }
+						{ footer && (
+							<p className="googlesitekit-publisher-win__footer-text">
+								{ footer }
+							</p>
```

Comment:
I think if we go with this approach, then it should be a `<div>` tag instead of `<p>` because in the future we might want to use it somewhere else with a footer that consists of different block-level elements which can't be put within `<p>` tag.

That's said we need to update this to be something like `<div className="googlesitekit-publisher-win__footer">{ footer }</div>` and then in the `SetupBanner` we need to pass `<p className="...">...</p>` to the `footer` prop of the `BannerNotification` element. It also means that we need to move styles for the `.googlesitekit-publisher-win__footer-text` to be specific only for the `SetupBanner` component.